frontend/packages/kubevirt-plugin/src/plugin.tsx

```diff
@@ -191,6 +193,19 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: FLAG_KUBEVIRT,
     },
   },
+  {
+    type: 'Dashboards/Storage/Capacity/Dropdown/Item',
+    properties: {
+      metric: 'VMs vs Pods',
+      queries: {
+        'VMs vs Pods': [
```

Comment:
why is this object and not array ? seems like the key is necessary as it is the same as `metric` ?